### DISCUSSION TITLE
REDUCE is being detected as Mathematica/Red/MATLAB.

### DISCUSSION THREAD

**Discussion by johnsonjh** [github-linguist/linguist#5238] [Classification] [answered]
REDUCE is being (incorrectly) detected as a mix of Mathematica, Red, and MATLAB:

* [Mathematica](https://github.com/reduce-algebra/reduce-algebra/search?l=mathematica) in [reduce-algebra](https://github.com/reduce-algebra/reduce-algebra)
* [Red](https://github.com/reduce-algebra/reduce-algebra/search?l=red) in [reduce-algebra](https://github.com/reduce-algebra/reduce-algebra)
* [MATLAB](https://github.com/reduce-algebra/reduce-algebra/search?l=matlab) in [reduce-algebra](https://github.com/reduce-algebra/reduce-algebra)

**Comment by Nixinova** [comment, marked as answer]
That's because reduce isn't listed as a language known to Linguist. If it is used in 200+ repos then reduce can be added as it's own language.

**Reply by johnsonjh** [reply]
Thanks for the clarification.
